src/main/webapp/app/exercises/programming/shared/instructions-render/task/programming-exercise-instruction-task-status.component.ts

```diff
@@ -24,12 +24,15 @@ export class ProgrammingExerciseInstructionTaskStatusComponent {
     @Input() exercise: Exercise;
     @Input() latestResult?: Result;
 
-    testsValue: string[];
+    /**
+     * array of test ids
+     */
+    testsValue: number[];
```

Comment:
rename this `testIds` and remove comment above?

Reply:
I kept the comment since the value is an input from the outside.